hack/cmd_util.sh

```diff
@@ -182,16 +184,18 @@ function os::cmd::internal::expect_exit_code_run_grep() {
 	done
 
 	if (( cmd_succeeded && test_succeeded )); then
-		os::text::print_green "SUCCESS after ${time_elapsed}s: ${name}"
+		os::text::print_green "SUCCESS after ${time_elapsed}s: ${name}" | tee "{JUNIT_REPORT_OUTPUT}"
```

Comment:
Someone is going to call this method without `JUNIT_RESPORT_OUTPUT` set.  Make this command work as expected for them and add a test to the tests for this file to prove that it works.